tasks/templates/tasks/session_list.html

```diff
@@ -0,0 +1,44 @@
+{% load static %}
+{% load i18n %}
+
+{% for session in study.session_set.all %}
+    <div class="task_list mt-4 mb-4">
+        <h3 class="mt-2">
+            {% blocktrans with order=session.order duration=session.tasks_duration repeats=session.repeats trimmed %}
+                Sessie {{ order }}
+            {% endblocktrans %}
+            {% if can_edit_sessions %}
+                <a href="{% url 'tasks:session_end' session.pk %}"
+                   class="text-decoration-none">
+                    <img src="{% static 'proposals/images/pencil.png' %}"
+                         title="{% trans 'Sessie aanpassen' %}">
+                </a>
+                <a href="{% url 'tasks:session_delete' session.pk %}"
+                   class="text-decoration-none">
+                    <img src="{% static 'proposals/images/delete.png' %}"
+                         title="{% trans 'Sessie verwijderen' %}">
+                </a>
+            {% endif %}
+        </h3>
+        <p>
+            <em>
+                {% blocktrans with duration=session.tasks_duration repeats=session.repeats trimmed %}
+                    Totale brutoduur: {{ duration }}
```

Comment:
Right now this will show "None" instead "Onbekend" when no duration is yet known. 

One way to solve this is to write a two-line template tag that takes a variable, and just replaces a None value with a translatable "Onbekend".